packages/server/src/api/routes/tests/queries/mongodb.spec.ts

```diff
@@ -52,25 +55,25 @@ describe("/queries", () => {
   ): Promise<T> {
     return await withClient(async client => {
       const db = client.db(
-        (await databaseTestProviders.mongodb.datasource()).config!.db
+        (await getDatasource(DatabaseName.MONGODB)).config!.db
       )
       return await callback(db.collection(collection))
     })
   }
 
   afterAll(async () => {
-    await databaseTestProviders.mongodb.stop()
     setup.afterAll()
   })
 
   beforeAll(async () => {
     await config.init()
     datasource = await config.api.datasource.create(
-      await databaseTestProviders.mongodb.datasource()
+      await getDatasource(DatabaseName.MONGODB)
     )
   })
 
   beforeEach(async () => {
+    collection = generator.guid()
```

Comment:
Because tests are reused between runs now, we can't use hard-coded table/collection names, so we generate unique ones for each test run.